bin/start-syncel-deploy.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -euo pipefail
+
+version="${1:-$version}"
+fesHostArr=("fes1" "fes2")
+allDeployHosts="cel1 cel2 cel3"
+IFS=' '
+read -a allDeployHostArr <<< "${allDeployHosts}"
+
+for deployHost in "${allDeployHostArr[@]}";
```

Comment:
no need for `read` since bash `for` word splits on space by default already.
```suggestion
allDeployHosts="cel1 cel2 cel3"

for deployHost in $allDeployHosts;
```